README.md

```diff
@@ -154,104 +154,108 @@ This table is generated from https://github.com/serverless/plugins/blob/master/p
 -->
 | Plugin | Author |
 |:-------|:------:|
-| **[Raml Serverless](https://github.com/andrewcurioso/raml-serverless)** <br/> Serverless plugin to work with RAML API spec documents | [andrewcurioso](http://github.com/andrewcurioso) |
```

Comment:
Any idea why is this updated? 🤔

Reply:
I ran `npm run docs`. I figured updating the docs required this. I can roll back if needed.